sdk/iot/azure-iot-modelsrepository/azure/iot/modelsrepository/__init__.py

```diff
@@ -0,0 +1,18 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+# Main Client
+from .client import ModelsRepositoryClient
+
```

Comment:
Given the recent pivot in the C# SDK to provide utility functions, it would be good to expose`is_valid_dtmi()` and `get_model_path()` functionality. Not directly part of the client.